src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -101,23 +115,46 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String conversationTitle = request.getParameter("conversationTitle");
     if (!conversationTitle.matches("[\\w*]*")) {
-      request.setAttribute("error", "Please enter only letters and numbers.");
-      request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
-      return;
+      if (conversationTitle.contains("#")) {
```

Comment:
let's create a boolean variable for this at the method level
i.e. boolean hasHashtag = conversationTitle.contains("#");
that way, we don't have to do this expensive check so many times